site/content/contribute/mobile/developer-setup.md

```diff
@@ -210,3 +210,22 @@ xcrun: error: unable to find utility "instruments", not a developer tool or in P
   ![](/img/xcode_preferences.png)
 
   - After this go back to the command line and run ``make run-ios`` again.
+
+### Errors When Running react-native packager
+
+##### Error message
+```sh
+FATAL ERROR: CALL_AND_RETRY_LAST Allocation failed - JavaScript heap out of memory
+```
+
+##### Solution
+-   Increase `max_old_space_size` of the nodejs instance.  On Mac, add the following line to your `~/.bash_profile` file:
```

Comment:
Propose `nodejs` to `NodeJS`

Reply:
Propose `Mac` to `macOS`.

Reply:
Is this line specific for Mac? Or does it also apply for other platforms?

Reply:
I've encountered this on Mac but not sure same fix can be applied to other platform.

Reply:
Makes sense. How about we formatted it as follows:

````
- Increase `max_old_space_size` of the NodeJS instance.
  - On macOS, add the following line to your `~/.bash_profile` file: `export NODE_OPTIONS=--max_old_space_size=12000`

- Then reload your bash configuration:

    ```sh
    source ~/.bash_profile
    ```
````

That way increasing `max_old_space_size` is a general solution, and the command is a tip specifically for Mac

Reply:
Thanks @jasonblais, now updated.  Please see screenshot:
![screen shot 2018-12-03 at 1 34 03 pm](https://user-images.githubusercontent.com/5334504/49355559-22773380-f703-11e8-9af3-7700c121a53d.png)